cranelift/isle/isle/src/ast.rs

```diff
@@ -32,10 +32,7 @@ pub struct Ident(pub String, pub Pos);
 
 /// Pragmas parsed with the `(pragma <ident>)` syntax.
 #[derive(Clone, PartialEq, Eq, Debug)]
-pub enum Pragma {
-    /// Enable overlap errors in the source.
-    OverlapErrors,
-}
+pub enum Pragma {}
```

Comment:
Can we put a comment here stating that currently no pragmas are defined (but the infrastructure remains from past pragmas and we are choosing to retain it to easily support future ones)? Otherwise this might look a bit odd to a new reader with no context.